verible/verilog/analysis/checkers/constraint-name-style-rule.h

```diff
@@ -26,22 +30,36 @@
 namespace verilog {
 namespace analysis {
 
-// ConstraintNameStyleRule check each constraint name follows the correct naming
-// convention.
-// The constraints should be named with lower_snake_case and end with _c.
+// Lower snake case, ends with `_c`
+#define kSuffix "([a-z0-9]+_)+c"
+
+// ConstraintNameStyleRule checks that each constraint name follows the
+// specified naming convention.
+//
+// This convention is set by providing a regular expression to be matched
+// against.
+//
+// The default, `kSuffix` checks that the name is written in lower_snake_case
+// and ends with `_c`
 class ConstraintNameStyleRule : public verible::SyntaxTreeLintRule {
  public:
   using rule_type = verible::SyntaxTreeLintRule;
 
   static const LintRuleDescriptor &GetDescriptor();
 
+  absl::Status Configure(absl::string_view configuration) final;
   void HandleSymbol(const verible::Symbol &symbol,
                     const verible::SyntaxTreeContext &context) final;
 
   verible::LintRuleStatus Report() const final;
 
+  std::string Pattern() const { return regex->pattern(); }
```

Comment:
Is this public method (returning a string by value) needed anywhere ?

Reply:
It isn't, I used it to diagnose if the user-provided regex failed to parse but it is not needed anymore thanks to the `SetRegex` utility. Missed it, thanks!